lib/components/Input/base/index.tsx

```diff
@@ -127,11 +129,8 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
                             {rightIcon}
                         </span>
                     )}
-                    {triggerActionIcon && (
-                        <>
-                            {triggerActionIcon}
-                        </>
-                    )}
+                    {dropdownIcon && dropdownIcon}
```

Comment:
btw you can pass dropdown Icon into triggerActionIcon prop. I made the name generic so any icon can be passed in. No need to define a new prop 👍

Reply:
Thanks @henry-deriv , i'll update it